challenges/2021_CyrilMorcrette_cloudBaseHeight/cbh_challenge_send_data_to_zarr.ipynb

```diff
@@ -0,0 +1,2569 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    #if duplicate cubes exist, concatenate them using numpy to avoid metadata matching issues
</pre><p>You could porobably make this sort of thing a part of the previous markdown cell, rather than code comments. That's ust a stylistic thing though, I find the markdiown easier to read, you have a different opinion.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MetOffice/data_science_cop/pull/30/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>